src/wrappers/ActionHandlingComponentWrapper.tsx

```diff
@@ -12,13 +11,13 @@ export class ActionHandlingComponentWrapper
   extends ComponentWrapper
   implements AdyenActionComponent
 {
+  constructor(nativeModule: NativeModule, events?: Event[]) {
+    var allEvents = [Event.onAdditionalDetails];
```

Comment:
It should be `const`

Reply:
Fixed in [9333374](https://github.com/Adyen/adyen-react-native/pull/599/commits/9333374bfb1fb058e7115d9134c73000082fddc9)